qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -389,44 +390,41 @@ def convert_acquire(self, instruction):
         t0 = instruction.t0
         duration = instruction.duration
         qubits = instruction.qubits
-        discriminators = (instruction.discriminators
-                          if hasattr(instruction, 'discriminators') else None)
+        qubit_channels = [channels.AcquireChannel(qubit, buffer=self.buffer) for qubit in qubits]
 
-        kernels = (instruction.kernels
-                   if hasattr(instruction, 'kernels') else None)
+        mem_slots = [channels.MemorySlot(instruction.memory_slot[i]) for i in range(len(qubits))]
 
-        mem_slots = instruction.memory_slot
-        reg_slots = (instruction.register_slot
-                     if hasattr(instruction, 'register_slot') else None)
+        if hasattr(instruction, 'register_slot'):
+            register_slots = [channels.RegisterSlot(instruction.register_slot[i])
+                              for i in range(len(qubits))]
+        else:
+            register_slots = None
 
+        discriminators = (instruction.discriminators
+                          if hasattr(instruction, 'discriminators') else None)
         if not isinstance(discriminators, list):
-            discriminators = [discriminators for _ in range(len(qubits))]
+            discriminators = [discriminators]
+        if any(discriminators[i] != discriminators[0] for i in range(len(discriminators))):
+            warnings.warn("Can currently only support one discriminator per acquire.")
```

Comment:
Could you please add something like. "Can currently only support one discriminator per acquire. Defaulting to first discriminator entry." To make the behaviour clear.